tests/test_page_mixin_model.py

```diff
@@ -95,6 +95,13 @@ def test_newsletter_html_get_template():
     assert page.get_newsletter_template.call_count == 1
 
 
+def test_preview_default():
+    page = ArticlePage(title="Page title")
+    request = RequestFactory().get("/")
+    response = page.serve_preview(request, "default").render()  # type: ignore
+    assert '<h1 class="web">Page title</h1>' in response.content.decode()
```

Comment:
minor comment, feel free to ignore: in simple cases like this we can make the test _a tiny bit_ faster by not calling `.decode()`, and use a bytes literal: 

```suggestion
    assert b'<h1 class="web">Page title</h1>' in response.content
```

But that's clearly a micro-optimisation that doesn't really matter as this codebase will likely keep a modest size in the long term, with a rather fast test suite, so of course feel free to ignore :slightly_smiling_face:

Reply:
I like it because it's shorter :)